packages/expo-notifications/README.md

```diff
@@ -1,5 +1,115 @@
 # expo-notifications
 
-`expo-notifications` provides some great stuff.
+## Instalation 
+
+`expo-notifications` needs `react-native-unimodule` to work. You can read more about it [here](https://www.npmjs.com/package/react-native-unimodules). 
+
+Add `expo-notifications` to your project:
+execute the following command in your project root directory:
+"yarn add expo-notifications".
+
+### Android
+
+#### 1. Modify AndroidManifest
+
+`AndroidManifest.xml` is a place where you can configure `expo-notifications` module. For Instance, you can choose there
+if you want to use Expo's servers as a middleman or choose that you don't want to use push notifications at all.
+
+Configuration template:
+
+```xml
+
+<!-- expo-notifications START -->
+    <!-- turn on push notifications START -->
+        <service
```

Comment:
I think you should show whole Manifest hierarchy and service place within it. E.g.:
```xml
<manifest ...>
  <application
   ...
  >
    <!-- Your example service -->
  </application>
</manifest>
```

Some unexperienced Android developers might find it not so obvious.